examples/gif_paintable/gif_paintable/frame.rs

```diff
@@ -0,0 +1,53 @@
+use std::time::Duration;
+
+use gtk::prelude::*;
+use gtk::{gdk, glib};
+
+pub struct Frame {
+    pub width: i32,
+    pub height: i32,
+    pub texture: gdk::Paintable,
```

Comment:
Store it as a `gdk::Texture` and you can get width/height from it instead of storing it separately